### ISSUE TITLE
dhparam needs quieting option for scripting

### ISSUE THREAD

**Issue by pprindeville** [openssl/openssl#17285]
I'm using a `/etc/cron.daily/` file that does:

```
#!/bin/sh

BITS=4096
GENERATOR=-2

openssl dhparam -out /etc/pki/tls/misc/dhparam.pem.new \
		$GENERATOR $BITS > /dev/null \
	&& mv -f /etc/pki/tls/misc/dhparam.pem.new /etc/pki/tls/misc/dhparam \
	&& apachectl restart
```

to generate daily `dhparam` files which I then use in `ssl.conf` as:

```
SSLOpenSSLConfCmd DHParameters /etc/pki/tls/misc/dhparam.pem
```

to change my key exchange parameters daily.  Problem is that:

1. `cron` gets confused by commands writing to `stderr` even if the script exits with `0` and wants to report it as a possible failure;
2. more broadly, this violates the UNIX principle of "be quiet if everything succeeds, but complain if it doesn't" (i.e. "no news is good news");

Other commands that are likely to be batched or at least scripted (like `genrsa` or `ecparam`) would do well to either send "busy noise" (as opposed to errors) to `stdout` or have an `openssl -q <cmd> ...` option to stifle output in normal execution.

**Comment by pprindeville** [comment]
See PR #17334.

**Comment by pprindeville** [comment]
Changed from `-quiet` to `-verbose` to be consistent with `req`, `genrsa`, `dsaparam`, `gendsa`, etc.

**Comment by t8m** [comment]
Fixed by #17336
